src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -64,31 +75,66 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<Entity> events =
         new ArrayList<Entity>(results.asList(FetchOptions.Builder.withDefaults()));
 
-    // get location
-    // filter by location and cutoff outside it
+    // Get location of user
+    String location = request.getParameter("location");
+    LatLng userLocation = getLatLng(location);
+
+    // Get distance between user and the location of all the events
+    for (Entity event : events) {
+      LatLng eventLocation =
+          getLatLng(
+              event.getProperty("streetAddress").toString()
```

Comment:
It's more readable to define an array of `locationProperties` and do `String.join(" ", locationProperties);`

Reply:
Got it, thanks!